psp/exp_configs/island.py

```diff
@@ -1,59 +1,64 @@
 """Main config for the "island" use-case."""
 
 import datetime as dt
-import functools
 
 from psp.data.data_sources.nwp import NwpDataSource
 from psp.data.data_sources.pv import NetcdfPvDataSource, PvDataSource
-from psp.dataset import Splits, split_train_test
+from psp.dataset import DateSplits, PvSplits, auto_date_split, split_pvs
 from psp.exp_configs.base import ExpConfigBase
 from psp.models.base import PvSiteModel, PvSiteModelConfig
 from psp.models.recent_history import RecentHistoryModel
 from psp.models.regressors.decision_trees import SklearnRegressor
 from psp.typings import Horizons
 
-PV_DATA_PATH = "data/island/data_15min_FIN.nc"
-NWP_PATH = "data/island/nwp_v6.nwp"
+PV_TARGET_DATA_PATH = "data/island/data_hourly_MW_dstfix_clean_v2.nc"
+NWP_PATH = "data/island/nwp_v8.zarr"
 
 
 class ExpConfig(ExpConfigBase):
-    @functools.cache
     def get_pv_data_source(self):
         return NetcdfPvDataSource(
-            PV_DATA_PATH,
+            PV_TARGET_DATA_PATH,
             id_dim_name="id",
             timestamp_dim_name="datetimeUTC",
             rename={
-                "Power at point in time MW": "power",
-                "Total Installed Capacity MWp": "capacity",
+                "Hourly PV Generated Units (MW)": "power",
+                "Total Max Capacity (MW)": "capacity",
             },
         )
 
-    @functools.cache
     def get_data_source_kwargs(self):
         return dict(
-            pv_data_source=self.get_pv_data_source(),
+            pv_data_source=NetcdfPvDataSource(
+                PV_TARGET_DATA_PATH,
+                id_dim_name="id",
+                timestamp_dim_name="datetimeUTC",
+                rename={
```

Comment:
are you not renaming twice?

Reply:
2 different data sources: one for the targets, and one for the features for the model. They have to be different because they don't have the same blackout (lag).